### ISSUE TITLE
[java] AssertJ SimplifyStartsWith false-positive

### ISSUE THREAD

**Issue by pnsantos** [pmd/pmd#2712]
**Affects PMD Version:** 6.26

**Rule:** [SimplifyStartsWith](https://pmd.github.io/latest/pmd_rules_java_performance.html#simplifystartswith)

**Description:**

When using assertJ like so:
```java
assertThat(someStringValue).startsWith("a");
```
PMD is flagging this with:
```
Rule:SimplifyStartsWith Priority:3 This call to String.startsWith can be rewritten using String.charAt(0).
```

This feels like it's a false-positive, even though one could make it be:
```java
assertThat(someStringValue.charAt(0)).isEqualTo("a");
```
it feels much less idiomatic use of the library's API

**Comment by adangel** [comment]
Fixed via #2899 for PMD 7.
